### DISCUSSION TITLE
Is there a way to toggle background image?

### DISCUSSION THREAD

**Discussion by jyyhuang** [wezterm/wezterm#5297] [Q&A] [answered]
### What Operating System(s) are you running on?

Windows

### WezTerm version

20240405-180910-cce0706b

### Ask your question!

I wanted to create a key bind to toggle on and off my background image but couldn't figure out how to do this. How can I do this?

**Comment by nekowinston** [comment, marked as answer]
See the example in https://wezfurlong.org/wezterm/config/lua/window/set_config_overrides.html
